pkg/controller/releasemanager/incarnation.go

```diff
@@ -20,6 +20,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	externalTestTimeoutTime = 10 * time.Minute
```

Comment:
Do you think it makes sense to make this a default and allow users to set the timeout? I can imagine different apps having different test periods.